SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorSettings.cpp

```diff
@@ -120,6 +120,21 @@ void USpatialGDKEditorSettings::PostEditChangeProperty(struct FPropertyChangedEv
 			return;
 		}
 	}
+	else if (Name == GET_MEMBER_NAME_CHECKED(USpatialGDKEditorSettings, LaunchConfigDesc))
+	{
+		if (PropertyChangedEvent.Property->GetFName() == GET_MEMBER_NAME_CHECKED(FWorldLaunchSection, LegacyFlags))
```

Comment:
Is it also worth trimming the values here? Do you know what happens if a runtime flag is set to "true "?

Reply:
Setting some LegacyFlags to `true ` or `    360000    ` doesn't seem to break any deployment. So it might not be worth it, but doesn't cost much either. Should I trim them?

Reply:
Danny took a look at the parsing on squid's side, and some values would have been invalid such as `True ` not being accepted. 
So I added the value trimming